data/covering_grammar/lib/make_test_file.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Makes test file.
+
+Takes the gold data and the model output, and creates a three-column TSV where
+each line has a word, its gold pronunciation, and the predicted pronunciation.
+Assumes that the input files have the same words in the same order.
+"""
+
+import argparse
+import logging
+
+
+def main(args: argparse.Namespace) -> None:
+    with open(args.gold, "r") as gf, open(args.pred, "r") as pf:
+        with open(args.out, "w") as wf:
+            for i, (g_line, p_line) in enumerate(zip(gf, pf)):
+                try:
+                    g_data = g_line.split("\t")
+                    p_data = p_line.split("\t")
+                    # Make sure that gold data and predictions have the
+                    # same words
+                    assert g_data[0] == p_data[0]
+                    word = g_data[0]
+                    # Note that we use `strip` to remove the newline
+                    g_pron = g_data[1].strip()
+                    p_pron = p_data[1].strip()
+                    line = "\t".join([word, g_pron, p_pron])
```

Comment:
I'd just do `line = f"{word}\t{g_pron}\t{p_pron}"`; I think that's easier to read.